Automattic/wp-calypso
# Modularize Launch Site and add Launch Site button to Site Tools

Fixes https://github.com/Automattic/dotcom-forge/issues/5324

## Proposed Changes

* Modularize Launch Site Component
* Integrate into `/settings/general/:site`
* Integrate into `/settings/site-tools/:site`

## Testing Instructions



* Have a site that is unlaunched
* Go to `/settings/general/:site` and test SIte launch
* After site launch, should show privacy settings
* Prepare another site that is unlaunched
* Go to `/settings/site-tools/:site` and test Site launch
* After site launch, launch site component doesn't show

## Pre-merge Checklist
